### ISSUE TITLE
Expo url setup for nhost-cli local development and production use cases

### ISSUE THREAD

**Issue by Svarto** [nhost/cli#48]
The dockers are set up to serve on localhost at the moment, this makes it not possible to connect to them over the network from anotehr computer, or in my specific case during RN Expo development, through the Expo client which connects over the local network to the services.

If the option existed to choose to serve the dockers to listen on 0.0.0.0 or the current local network IP, then this would allow for reaching the services over the local LAN.

**Comment by Svarto** [comment]
For anyone coming after me, even on MacOS Docker actually exposes the ports correctly and if Mac Firewall is off the services are accessible from the local network (i.e. so that Expo Go can point to the local dev environment from a physical device).

For Expo, you can set the URLs as below, this makes the production builds point to the nhost cloud (you need to insert your URL here), the physical devices using Expo Go point to the IP of the machine running the Metro Bundler and the simulators run on the machine point to localhost. 

TLDR: In Expo, to make development use the dev environment and production use production, see code below:

```
export const auth_url =
  Constants.manifest.packagerOpts.dev && !Constants.isDevice
    ? "http://localhost:9001"
    : typeof manifest.packagerOpts === `object` && manifest.packagerOpts.dev
    ? `http://${manifest.debuggerHost.split(`:`).shift().concat(`:9001`)}`
    : "nhost.io cloud url";
```

**Comment by Svarto** [comment]
@nunopato @elitan this could benefit from being added to the Nhost docs for future Expo users. Closing the issue here for now though as it is solved.

**Comment by elitan** [comment]
> @nunopato @elitan this could benefit from being added to the Nhost docs for future Expo users. Closing the issue here for now though as it is solved.

Could you open an issue for that at https://github.com/nhost/docs ?
